CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Changed
 
+* Annotate Service objects by load-balancer UUIDs to enable free LB renames and improve the DO API consumption performance (@timoreimann)
```

Comment:
Just noticed this got merged into the `v0.1.16` section

Reply:
Good catch -- submitted #249 to fix.